modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/common/Constants.java

```diff
@@ -289,8 +289,12 @@ private AndroidConfigurationManagement() {
         public static final String ROLE_NAME = "administration";
         public static final String UPDATED_ROLE_NAME = "manager";
         public static final String ROLE_ENDPOINT = "/api/device-mgt/v1.0/roles";
+        public static final String ROLE_ENDPOINT_WITH_PAGINATION
+                = "/api/device-mgt/v1.0/roles?offset=0&limit=2";
         public static final String SCOPE_ROLE_ENDPOINT = "/api/device-mgt/v1.0/roles/scopes";
         public static final String ROLE_PAYLOAD_FILE_NAME = "role-payloads.json";
+        public static final String ROLE_RESPONSE_PAYLOAD
+                = "[{\"key\":\"role:view\",\"name\":\"View roles\",\"roles\":\"admin,manager\",\"description\":\"\"},{\"key\":\"role:manage\",\"name\":\"Add roles\",\"roles\":\"admin,manager\",\"description\":\"\"}]";
```

Comment:
Thank you for the suggestion. This constant is no longer in use. Therefore I have removed this. 
https://github.com/wso2/product-emm/pull/368